transform-plugins/src/main/java/co/cask/hydrator/plugin/CSVParser.java

```diff
@@ -144,9 +149,26 @@ Schema parseAndValidateOutputSchema(Schema inputSchema) {
     }
   }
 
+  @Override
+  public void prepareRun(StageSubmitterContext context) throws Exception {
+    super.prepareRun(context);
+    init();
+    if (fields != null) {
+      FieldOperation operation = new FieldTransformOperation("CSV Parse", "Parsing the field.",
```

Comment:
do we have any guidelines or thinking around how we want to name these operations. For example, why 'CSV Parse' instead of just 'Parse', why 'Read' instead of 'TPFS Read' or 'TPFS Avro Read'.

Reply:
using `Parse`, `Read` etc. rather than adding the plugin name.